pygdf/dataframe.py

```diff
@@ -594,6 +594,144 @@ def _n_largest_or_smallest(self, method, n, columns, keep):
                 df[k] = self[k].reset_index().take(new_positions)
         return df.set_index(self.index.take(new_positions))
 
+    def merge(self, other, on=None, how='multi-left', lsuffix='_x', rsuffix='_y'):
```

Comment:
`how` should not expose "multi-left" to be more compatible with pandas.  Also, the `test_multi.py::test_dataframe_multi_column_join` is failing because it is passing in `left`.

Reply:
@kkraus14, I agree.  When the parameters do overlap with pandas, we should keep them compatible.  But, it is okay to add new parameters that pandas do not have.